build/prepare_restored_cluster.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+set -o xtrace
+set -o errexit
+
+function escape_special() {
+	{ set +x; } 2>/dev/null
+	echo "$1" |
+		sed 's/\\/\\\\/g' |
+		sed 's/'\''/'\\\\\''/g' |
```

Comment:
If we run `shfmt ` locally for this specific file, these will be resolved, given that it is a new file and not a legacy one.